packages/core/testing/cable/index.js

```diff
@@ -0,0 +1,29 @@
+export class TestCable {
+  constructor() {
+    this.outgoing = []
+    this.channels = {}
+  }
+
+  async subscribe(channel) {
+    channel.connecting(this)
+
+    this.channels[channel.identifier] = channel
+    channel.connected(channel.identifier)
+
+    return Promise.resolve(channel.identifier)
+  }
+
+  async perform(identifier, action, payload = {}) {
+    this.outgoing.push(payload)
```

Comment:
> should outgoing array stores action name or only payload (In the example above I see only payload)

Yeah, I think we need actions as well. Maybe, we can use some interface like:

```ts
type PerformedAction = {action: string, payload: object}
this.outgoing.push({action, payload}}
```

Reply:
Added an action for outgoing array. Also added type